src/setup.py

```diff
@@ -19,7 +19,7 @@
 
 setup(
     name='katello-agent',
-    version='3.5.7',
+    version='3.5.8',
     description='Katello Agent',
     author='Jeff Ortel',
     author_email='jortel@redhat.com',
```

Comment:
While we're at it, should this also be updated?

Reply:
Must author and author_email be specified? There is no singular author or point of contact, so if I can remove them safely then that'd be good

Reply:
https://python-packaging.readthedocs.io/en/latest/minimal.html suggests it does. If you remove it, you get this:

> warning: check: missing meta-data: either (author and author_email) or (maintainer and maintainer_email) should be supplied

As the author you could say something like `The Foreman Project` but not sure what the email should be. Perhaps something generic as `no-email@theforeman.org`?